x-pack/plugin/esql/qa/testFixtures/src/main/resources/mv_median_absolute_deviation.csv-spec

```diff
@@ -0,0 +1,82 @@
+example
+required_capability: fn_mv_median_absolute_deviation
+
+// tag::example[]
+ROW values = [0, 2, 5, 6]
+| EVAL median_absolute_deviation = MV_MEDIAN_ABSOLUTE_DEVIATION(values), median = MV_MEDIAN(values)
+// end::example[]
+;
+
+// tag::example-result[]
+values:integer | median_absolute_deviation:integer | median:integer
+[0, 2, 5, 6]   | 2                                 | 3
+// end::example-result[]
+;
+
+fromIndex
+required_capability: fn_mv_median_absolute_deviation
+
+FROM employees
+| WHERE emp_no <= 10002
```

Comment:
This interval has a better coverage, it's also covering `null`.

```suggestion
| WHERE emp_no <= 10010
```

Reply:
Added the emp_no 10009 (null) and 10007 (odd amount of values). The others are identical, so just those to keep the test simple